src/Home.tsx

```diff
@@ -159,42 +150,32 @@ const Home = (props: HomeProps) => {
     })();
   }, [wallet, props.connection]);
 
-  useEffect(refreshCandyMachineState, [
-    wallet,
-    props.candyMachineId,
-    props.connection,
-  ]);
+  useEffect(refreshCandyMachineState, [wallet, props.candyMachineId, props.connection]);
 
   return (
-    <main>
-      {wallet && (
-        <p>Wallet {shortenAddress(wallet.publicKey.toBase58() || "")}</p>
-      )}
-
+    <div>
```

Comment:
I think we need to do something about this section. The light text on the honeycomb background is hard to read. Could we go with a different font color or maybe throw a shadowbox behind i
![Screen Shot 2021-11-03 at 11 12 23 PM](https://user-images.githubusercontent.com/6373522/140251782-de2c9edf-a41b-4e57-957a-622bcd14e726.png)
t.

Reply:
@lamar-frankie 
Totally agree, i'll see if I can figure something out. totally open to suggestions btw. 

Also, this is currently a `.png` file that I want to convert over to `svg` - hoping that will address the issue with responsiveness.